src/index.ts

```diff
@@ -138,6 +139,7 @@ export interface SpannerOptions extends GrpcClientOptions {
   sslCreds?: grpc.ChannelCredentials;
   routeToLeaderEnabled?: boolean;
   directedReadOptions?: google.spanner.v1.IDirectedReadOptions | null;
+  observabilityOptions?: ObservabilityOptions;
```

Comment:
Add documentation for observabilityOptions above interface similar to other options

Reply:
Done, thank you @surbhigarg92!